engineering/documents/CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+# Contributing Guidelines
+
+source{d} go-git project is [Apache 2.0 licensed](LICENSE) and accept
+contributions via GitHub pull requests.  This document outlines some of the
+conventions on development workflow, commit message formatting, contact points
+and other resources to make it easier to get your contribution accepted.
+
+## Certificate of Origin
+
+By contributing to this project you agree to the Developer Certificate of
+Origin (DCO). This document was created by the Linux Kernel community and is a
+simple statement that you, as a contributor, have the legal right to make the
+contribution. See the [DCO](DCO) file for details.
+
+## Support Channels
+
+Whether you are a user or contributor, official support channels include:
+
+- GitHub [issues](https://github.com/src-d/go-git/issues/new)
+- Slack: #go-git room in the [source{d} Slack](http://sourced.tech/)
+
+Before opening a new issue or submitting a new pull request, it's helpful to
+search the project - it's likely that another user has already reported the
+issue you're facing, or it's a known issue that we're already aware of.
+
+
+## How to Contribute
+
+PRs are the main and exclusive way to contribute to the official go-git project.
```

Comment:
Spell the first time `PRs` as `Pull Requests (PRs)`, then later refer as PRs normally. Gives more clarity about what you're talking about in case the person is not used to the lingo.

Reply:
Shouldn't we mention the workflow/branch usage of the repository as well?

Reply:
yep, make sense